pkg/instrumentation/dotnet.go

```diff
@@ -127,7 +127,7 @@ func injectDotNetSDK(dotNetSpec v1alpha1.DotNet, pod corev1.Pod, index int, runt
 		pod.Spec.InitContainers = append(pod.Spec.InitContainers, corev1.Container{
 			Name:      dotnetInitContainerName,
 			Image:     dotNetSpec.Image,
-			Command:   []string{"cp", "-a", "/autoinstrumentation/.", dotnetInstrMountPath},
+			Command:   []string{"cp", "-r", "/autoinstrumentation/.", dotnetInstrMountPath},
```

Comment:
What *is* the ownership of these files after the change? Before, it was the owner of the file in the image, which depends on how it's built. Now, it's going to be the user we run the initcontainer as. Is that well defined?

Reply:
Yes. When I was fixing the instrumentation for other languages, I found the same problem and we fixed it just copying the folder without preserving the ownership -it is actually not needed-.